src/OpenTelemetry.Exporter.Geneva/CHANGELOG.md

```diff
@@ -2,10 +2,17 @@
 
 ## Unreleased
 
+* Fix a bug where metrics without exemplars were not getting exported.
+  ([#1099](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/1099))
+
 * Relaxed table name mapping validation rules to restore the previous behavior
   from version 1.3.0.
   ([#1109](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/issues/1109))
 
+* Add support for exporting metrics to more than a single account/namespace
```

Comment:
please mention here the name of dimension keys which should contain account and namespace.